examples/http/src/main/java/io/opentelemetry/example/http/HttpClient.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.example.http;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.context.propagation.HttpTextFormat;
+import io.opentelemetry.exporters.inmemory.InMemorySpanExporter;
+import io.opentelemetry.exporters.logging.LoggingExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.SpanData;
+import io.opentelemetry.sdk.trace.TracerSdkFactory;
+import io.opentelemetry.sdk.trace.export.SimpleSpansProcessor;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Status;
+import io.opentelemetry.trace.Tracer;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.charset.Charset;
+
+public class HttpClient {
+
+  static int port = 8080;
+  // OTel API
+  Tracer tracer = OpenTelemetry.getTracerFactory().get("io.opentelemetry.example.http.HttpClient");
+  // Export traces in memory
+  InMemorySpanExporter inMemexporter = InMemorySpanExporter.create();
```

Comment:
InMemory is only useful for testing, we should not have an example unless is a test with this.

Reply:
yes, the logging exporter is here for this purpose!

Reply:
I am fine with the logging (because user can see something printed), but I am suggesting to remove the inmemory.

Reply:
and I'm agreeing!  :)

Reply:
Initially, I decided to use the InMemory exporter to showcase how traces can be debugged without a collector. But I agree with both of you, the logging should suffice